symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/datafeed/RealTimeEventListener.java

```diff
@@ -11,6 +11,19 @@
 @API(status = API.Status.STABLE)
 public interface RealTimeEventListener {
 
+    /**
+     * Check if the event is accepted to be handled.
```

Comment:
Here we should document what the default behavior is and why you might want to override it.